src/controllers/v2/note.js

```diff
@@ -0,0 +1,55 @@
+import database from '../../models/index.js';
+
+const {Note} = database;
+
+const NoteController = {
+  /**
+   * The create function writes the incoming Note to the appropriate database tables.
+   *
+   * @param {any } appId The application that the Note will be based on.
+   * @param {any | undefined} incomingNote The Note details.
+   * @returns {any} Returns newNote, the newly created Note.
+   */
+  // prettier-ignore
```

Comment:
You don't need to disable this linting rule, or the other 3 `// prettier-ignore` exceptions you've added to this file.

Reply:
Done

Reply:
Done

Reply:
Resolved in https://github.com/Scottish-Natural-Heritage/standard-forestry-operations-api/pull/77/commits/e33b7df9f6afa48d4e7d2ef0bd9782a3a13aa654